extension/parquet/parquet_reader.cpp

```diff
@@ -71,6 +71,9 @@ static shared_ptr<ParquetFileMetadataCache> LoadMetadata(Allocator &allocator, F
 	transport.SetLocation(file_size - 8);
 	transport.read((uint8_t *)buf.ptr, 8);
 
+	if (memcmp(buf.ptr + 4, "PARE", 4) == 0) {
+		throw InvalidInputException("Encrypted Parquet files are not supported", file_handle.path);
```

Comment:
And another very minor one, not really practical here, you can consider having a structure like:
```c++
if ( str != "PAR1" ) {
      if (str == "PARE") {
            throw "encrypted not supported"
      } else {
            throw "no magic bytes";
      }
}
// rest of the function
```
It's minor, but this would allow a single comparison in the expected good case.

Reply:
Thanks voor pointing this out! It looks like that the exception thrown is actually a `FormatException()` from `ParquetReader::InitializeSchema()`. So the initial `InvalidInputException()` doesn't seem to work anyways. Maybe it is better to let it as it was or only keep the test case?

Reply:
> This exception seems to be off since it's has extra parameters not consumed by the format string.
> 
> Ideally this should be caught at the compiler or testing level, unsure why this is not the case here, but I guess fixing the general detection is out of scope for this PR.

This should be checked, code is in `exception.cpp:61`, it only runs in debug mode though, so that might be why it wasn't caught here